scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -367,6 +375,17 @@ case class Args(
       .map(_.stripPrefix(flag))
   }
 
+  def targetrootFromScalacOptions(): Option[String] =
+    dialect match {
```

Comment:
dialect should be used for parsing source only, I think we should switch on the `scalaVersion` instead

Reply:
ok

Reply:
I agree with you on this, but the scalaVersion can be empty and it's a little annoying.

Reply:
actually if we have the scalaVersion, the dialect is easy to deduce.